novawallet/Modules/AssetConversion/Service/AssetHub/AssetHubFeeService.swift

```diff
@@ -0,0 +1,345 @@
+import Foundation
+import RobinHood
+import BigInt
+
+final class AssetHubFeeService: AnyCancellableCleaning {
+    struct ChainOperationFactory {
+        let extrinsicOperationFactory: ExtrinsicOperationFactoryProtocol
+        let conversionOperationFactory: AssetConversionOperationFactoryProtocol
+        let conversionExtrinsicService: AssetConversionExtrinsicServiceProtocol
+    }
+
+    let wallet: MetaAccountModel
+    let chainRegistry: ChainRegistryProtocol
+    let operationQueue: OperationQueue
+
+    private var factories: ChainOperationFactory?
```

Comment:
why is it plural?

Reply:
because the Type holds multiple factories